### ISSUE TITLE
IndexError In section 5.1 of Stardist 3D

### ISSUE THREAD

**Issue by vineet-kureel** [HenriquesLab/ZeroCostDL4Mic#105]
Hi,
I am running google colab notebook for STARDIST 3D in my personal pc and it worked very well. But during the section 5.1 "Inspection of Loss Function" and getting an Index error and am unable to plot the loss over time. Quality Control step is bit crucial for me. I will be happy if you can help me with the problem.
The section fails to save the plot that is required for next section.

Thanks for making DL so easy for me :)

![bug](https://user-images.githubusercontent.com/67627842/120916525-c403e500-c6b2-11eb-8a78-dc077e54358c.png)

**Comment by guijacquemet** [comment]
Hi @vineet-kureel 
Thanks for reaching out! 
Unfortunately, I cannot reproduce the error on my end. 

Did your training complete until the end? 

Would you be able to share the folder containing your model?

Cheers

Guillaume

**Comment by vineet-kureel** [comment]
Hi @guijacquemet ,
Thanks for your fast response. Yes, the training was completed at the end. I have shared the folder containing my model with you.

https://drive.google.com/drive/folders/1JMC82CuEyWmO50vNk5jEnHDu7N5DkrX6?usp=sharing

Thank you, very much for your time.
